NEWS.md

```diff
@@ -1,18 +1,19 @@
 # opencage (development version)
 
-* The geocoding functions will not send a query to the API anymore if no API key is present (#133).
+* The geocoding functions will not send a query to the API anymore if no API key is present ([#133](https://github.com/ropensci/opencage/issues/133)).
```

Comment:
won't this be linked by pkgdown anyway? https://docs.ropensci.org/opencage/news/index.html

Reply:
Yes (but only there?!). Not on [CRAN](https://cran.r-project.org/web/packages/opencage/news/news.html) and [GH](https://github.com/ropensci/opencage/blob/main/NEWS.md) for example. 
😭 my [commit message](https://github.com/ropensci/opencage/commit/854eaa0b2e0a1d5891cc989c209ed45b0c2313bc) is wrong! 😉

Reply:
oh right. Could you build the website locally just to check this change won't mean double linking?

Reply:
It works for tidygeocoder ([NEWS](https://github.com/jessecambon/tidygeocoder/blob/main/NEWS.md), [pkgdown changelog](https://jessecambon.github.io/tidygeocoder/news/index.html)), so I assume it's going to be fine. 
Otherwise we can revert or file a bug report with pkgdown. 😁